app/controllers/katello/api/v2/host_packages_controller.rb

```diff
@@ -26,7 +27,7 @@ def index
       respond_for_index(:collection => collection)
     end
 
-    api :PUT, "/hosts/:host_id/packages/install", N_("Install packages remotely")
+    api :PUT, "/hosts/:host_id/packages/install", N_("Install packages remotely (deprecated)"), :deprecated => true
```

Comment:
Can you tell me what you have in mind as far as deprecating these APIs? I assume that it means we'll remove them, and so the command `hammer host package install` (among the others here) will be gone. It might be better to keep these APIs and delegate to remote execution, because it seems much more natural to type the former (you can guess it if you've used hammer a few times) vs `hammer job-invocation create --job-template 'Install Package - Katello SSH Default' --inputs=...`

We have another open PR in the spirit of keeping this API around, but supporting REX too: https://github.com/Katello/katello/pull/7678

Reply:
ahh thats an interesting idea.   Do you envision the UI using these APIs still?  Or would it redirect to the REX UI as it does today?

Reply:
regardless, updated to remove those api deprecation warnings, but am curious your thoughts on that

Reply:
Hmm good question. It's nice how the UI takes you to the job invocation details automatically via `remote_execution_controller`. We could remove the `remote_execution_controller` and provide the job invocation details in the API responses from this controller. UI can redirect based on that, or just provide notification with the link.